lsp/src/completion/tokens.rs

```diff
@@ -0,0 +1,65 @@
+use nom::{
+    branch::alt,
+    bytes::complete::{tag, take_while},
+    character::complete::{multispace0, satisfy},
+    combinator::{map, opt, recognize, value},
+    multi::many0,
+    sequence::{pair, preceded},
+    IResult, Parser,
+};
+
+#[derive(Debug, PartialEq, Clone)]
+pub enum Token {
+    Colon,
+    DoubleColon,
+    OpenAngle,
+    CloseAngle,
+    EqualSign,
+    OpenBracket,
+    ClosedBracket,
+    Identifier(String),
+    Jet,
+}
+
```

Comment:
For every parsing function please include string that is going to be accepted by the function